Development practices/Emails.md

```diff
@@ -0,0 +1,65 @@
+# Guidelines
+
+## Templates
+In most cases we don't need a custom design for the emails. We use [simple HTML templates](https://github.com/mailgun/transactional-email-templates) and modify them slightly with project's theme colors, and add a logo if needed.
```

Comment:
Slicing a basic custom email is not a problem, but the design and multiple iterations with the client are often very expensive compared to the value for the client.
But it makes sense to develop a few basic and clean templates which we could use instead of those I pasted in the chapter, as those look outdated. I'll open a task so we can discuss this.